src/sass/base/_va.scss

```diff
@@ -196,9 +196,11 @@ hr {
   clear: both;
   margin: 0;
   padding: 0;
+  min-height: 147.19px;
 
   @include media($medium-screen) {
     position: relative;
+    min-height: 168.5px;
```

Comment:
Will these min-heights still apply if we ever add content to the header in the future? Just wondering how these values were chosen.

Reply:
They'll have to be changed if we change the design of the header. I picked them because they're the final height in the browser, after everything has loaded. It's been pretty stable for a while, and doesn't affect content added in menus, so I don't think it'll cause much of a maintenance burden.